stepman/paths_test.go

```diff
@@ -0,0 +1,89 @@
+package stepman
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	GivenSteplibURI  = "https://github.com/bitrise-io/steplib"
+	GivenFolderAlias = "12334556"
+	GivenHomePath    = "/usr/testeruser/"
+	GivenStepID      = "test-custom-step"
+	GivenStepVersion = "0.5.6"
+)
+
+func GivenRoute() SteplibRoute {
+	return SteplibRoute{
+		SteplibURI:  GivenSteplibURI,
+		FolderAlias: GivenFolderAlias,
+	}
+}
+
+func Test_GivenHomeDir_WhenGetStepmanDirPathCalled_ThenGoodPathReturned(t *testing.T) {
+	// Given
+	err := os.Setenv("HOME", GivenHomePath)
+	require.NoError(t, err)
+	expected := path.Join(GivenHomePath, ".stepman")
+
+	// When
+	actual := GetStepmanDirPath()
+
+	// Then
+	assert.Equal(t, actual, expected)
+}
+
+func Test_GivenStepmanDir_WhenGetCollectionDirPathCalled_ThenGoodPathReturned(t *testing.T) {
+	// Given
+	err := os.Setenv("HOME", GivenHomePath)
+	require.NoError(t, err)
+	expected := path.Join(GetStepmanDirPath(), "step_collections")
+
+	// When
+	actual := GetCollectionsDirPath()
+
+	// Then
+	assert.Equal(t, actual, expected)
+}
+
+func Test_GivenRoute_WhenGetLibraryBaseDirPathCalled_ThenGoodPathReturned(t *testing.T) {
+	// Given
+	route := GivenRoute()
+	expected := path.Join(GetCollectionsDirPath(), route.FolderAlias, "collection")
+
+	// When
+	actual := GetLibraryBaseDirPath(route)
+
+	// Then
+	assert.Equal(t, expected, actual)
```

Comment:
🙈 Please be consistent with the order of `expected` and `actual`. One function above it has a different order then here. Sorry, just nitpicking.

Reply:
Nice catch! I'm always confused, Go does a lot of things the way around of what I'm used to 🙈